acq4/util/imaging/imaging_ctrl.py

```diff
@@ -151,8 +150,8 @@ def newFrame(self, frame):
 
     def saveFrameClicked(self):
         if self.ui.linkSavePinBtn.isChecked():
-            self.addPinnedFrame()
-        self.recordThread.saveFrame()
```

Comment:
Get all metadata fro controls at this point to make sure values are current and we don't try to access widgets from another thread